dirsrvtests/tests/suites/clu/dsconf_test.py

```diff
@@ -167,3 +167,36 @@ def test_dsconf_with_ldaps(topology_st, enable_config, config_type):
     msg = proc.communicate()
     log.info(f'output message : {msg[0]}')
     assert proc.returncode == 0
+
+@pytest.mark.parametrize('instance_role', ('consumer','hub'))
+def test_check_replica_id_rejected (topology_st, instance_role):
+    """Test dsconf CLI does not accept replica-id parameter for comsumer and hubs
+
+    :id: 274b47f8-111a-11ee-8321-98fa9ba19b65
+    :parametrized: yes
+    :customerscenario: True
+    :setup: 3 instance(s)
+    :steps:
+        1. Create ldap instance with 3 instances
+        2. Use dsconf cli to create replica and specify replica id for a consumer
+        3. Verify dsconf command rejects replica_id for consumers
+        4. Repeat for a hub use dsconf to create a replica w replica id
+        5. Verify dsconf command rejects replica_id for hubs
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Setting the "replica-id" manually for consumers not allowed.
+        4. Success
+        5. Setting the "replica-id" manually for hubs is not allowed.
+    """
+    print("DN_DM {}".format(DN_DM))
+    cmdline=['/usr/sbin/dsconf', 'standalone1', '-D',
+                 DN_DM, '-w', 'password', 'replication', 'enable', '--suffix',DEFAULT_SUFFIX,'--role',instance_role, '--replica-id=1' ]
```

Comment:
Please fix indentation and spacing around commas.  Perhaps run pep8 or flake to check the code syntax.

Reply:
Thank you! I fixed the indentation and I will also check any other syntax/formatting I could have missed.